beprof/curve.py

```diff
@@ -43,6 +45,7 @@ class Curve(np.ndarray):
     """
 
     def __new__(cls, input_array, **meta):
+        logging.info('Creating Curve object')
```

Comment:
Be more informative. Say sth about parameters of the method. It would be useful to know if `meta` is empty or what does it store